config/software/libarchive.rb

```diff
@@ -48,6 +48,7 @@
   env = with_standard_compiler_flags(with_embedded_path)
   update_config_guess(target: "build/autoconf/")
 
+  env["LDFLAGS"] << " -Wl,--disable-new-dtags"
```

Comment:
this could use an explanation in a comment or something, it looks weird.  "new dtags" are also very old at this point and its weird that we're needing to turn them off.